requirements.txt

```
@@ -1 +1,2 @@
-aiohttp>=3.7.0,<4.0
+aiohttp>=3.7.0,<4.0; python_version < "3.11"
+aiohttp>=3.8.2,<4.0; python_version >= "3.11"
```

```suggestion
aiohttp>=3.8.3,<4.0; python_version >= "3.11"
```
3.8.2 was yanked from pypi

---

But 3.8.2 does support 3.11; we shouldn't artificially block people from using a version unless *we're* known to be incompatiable with it.

---

I was mainly going by https://github.com/aio-libs/aiohttp/releases/tag/v3.8.3, which states
> Please, use `aiohttp ~= 3.8.3, != 3.8.1` instead, if you can.

But since pip won't use 3.8.2 unless you explicitly tell it to (`aiohttp==3.8.2`), I suppose it's fine then.